hypothesis/lms
# VitalSource API

# Testing notes

- Enable the VS flag on http://localhost:8001/flags
- Get the values from https://github.com/hypothesis/devdata/pull/28 (or `make devdata` once that's merged)
- Configure a vital-source assignment 
- Use `https://bookshelf.vitalsource.com/#/books/9781400847402` on the modal
- You should get the cover image and title and the ToC once you select the book

- Use a different URL with the same format  `https://bookshelf.vitalsource.com/#/books/9781400840000`
- You should get a `Book 9781400840000 not found` message in the UI
